evaluation.py

```diff
@@ -4,15 +4,17 @@
 import matplotlib.pyplot as plt
 import numpy as np
 import cv2
+import re
+import os
 
 class Evaluation():
 
-    def __init__(self, model_json, weights, df, num_hg_blocks, batch_size=1):
-        self.model_json = model_json              # json of model to be evaluated
-        self.weights = weights          # weights of model to be evaluated
-        self.df = df                    # df of the the annotations we want
-        self.num_hg_blocks = num_hg_blocks
-        self.batch_size = batch_size
+    def __init__(self, sub_dir, model_json, weights, h_net): 
+        self.model_json = os.path.join(sub_dir,model_json)         # json of model to be evaluated
+        self.weights = os.path.join(sub_dir,weights)          # weights of model to be evaluated
+        self.num_hg_blocks = int(re.match(r'.*stacks_(\d\d)_.*',model_json).group(1))
+        h_net._load_model(self.model_json, self.weights)
+        self.model = h_net.model
```

Comment:
Add sub dir for where to load model from. Grab num hg from model file name, store model as instance variable so it gets loaded once at `Evaluation` instance initialization.